client/state/posts/selectors.js

```diff
@@ -315,6 +316,23 @@ export function getPostEdits( state, siteId, postId ) {
 }
 
 /**
+ * Returns an object of normalized edited post attributes for the site ID post ID pairing.
+ *
+ * @param  {Object} state  Global state tree
+ * @param  {Number} siteId Site ID
+ * @param  {Number} postId Post ID
+ * @return {Object}        Normalized Post revisions
+ */
+export function getNormalizedPostEdits( state, siteId, postId ) {
+	const edits = getPostEdits( state, siteId, postId );
+	const normalize = flow( [
```

Comment:
See #6715 for how I'm proposing to move normalization flows to `./utils.js` and let me know what you think.